Detectors/ITSMFT/ITS/tracking/src/StandaloneDebugger.cxx

```diff
@@ -257,5 +258,21 @@ int StandaloneDebugger::getBinIndex(const float value, const int size, const flo
   return std::distance(divisions.begin(), TMath::BinarySearch(divisions.begin(), divisions.end(), value));
 }
 
+// Tracker
+void StandaloneDebugger::dumpTrackToBranchWithInfo(ROframe event, o2::its::TrackITSExt track,
+                                                   std::string branchName)
+{
+  // FakeTrackInfo t{event, track};
+
+  (*mTreeStream)
+    << branchName.data()
+    << track
```

Comment:
I would also an extended method to dump also the cluster, at least optionally, to be used in the fitting loop.